docs/human-interface-guidelines.md

```diff
@@ -323,7 +323,7 @@ Indicators are small icons that live in the menu bar (wingpanel). They give user
 
 ### Does Your App Need an Indicator? {#does-your-app-need-an-indicator}
 
-The indicator area is notorious for clutter and inconsistent behavior. Given that users will probably install many third-party apps, we must be careful about the number of indicators we show and how they behave. Keep in mind that not every application needs an indicator. Only a very small set of apps will benefit from one. You do not need an indicator if:
+The indicator area is prone to clutter and inconsistent paradigms. Given that users will probably install many third-party apps, we must be careful about the number of indicators we show and how they behave. Keep in mind that only a very small set of applications need or benefit from an indicator. Avoid adding an indicator if:
```

Comment:
I don't think using the word "paradigms" make this more clear. I'm okay with the other changes